### PR TITLE
New cloak added to the tg database

### PR DESCRIPTION
**Pull request by Comxy**
## About The Pull Request

I now know what github is all about, and want to improve the quality of our game as such. I have added it before to the game, but spawning it in is really easy and not rewarding. This is why, I added a cloak that rarely spawns into the maints and is craftable.
## Why It's Good For The Game

It benefits this game a whole lot. A lot of people have been playing for some time now. This drip cloak gives people something to search for, or a new objective to build a drip cloak. As you can see, making a drip cloak isn't the easiest thing to do. It will also be decently difficult to find it in the maints. That is why this would be a good new end-game like additions to our station.
## Changelog

:cl:
add: Added a new obtainable drip cloak found in maintenance or craftable.
/:cl:

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
I fear this PR's raw energy

**Comment by TheBonded** [comment]
If you wanted this for your own personal server, why not just merge this to your own split branch and run that code instead? It would just be a copy of tg's code but with the cloak added

**Comment by Nobelium-XVIII** [comment]
The longer he's on GitHub the more powerful he becomes...

**Comment by Comxy** [comment]
> 
> 
> If you wanted this for your own personal server, why not just merge this to your own split branch and run that code instead? It would just be a copy of tg's code but with the cloak added

There are three reasons I am trying to get this on the main branch. This is because:

1:
I don't play on my own server; I have never started up my own server, nor ever tried. To me seeing other people play or playing myself is to me way more fun. Also, I don't have much to test if I do start my own local server.

2:
This is a test thus maybe an excuse to add my own drip to the game. Everyone has to start somewhere with adding changes and this is a small thing, perfect for someone new to github branches. 

3:
I tried to add this with changing item icons, but the sad truth is that the icons are hardwired in the code using parent types.

**Comment by TheBonded** [comment]
> > 
> > 
> > If you wanted this for your own personal server, why not just merge this to your own split branch and run that code instead? It would just be a copy of tg's code but with the cloak added
> 
> There are three reasons I am trying to get this on the main branch. This is because:
> 
> 1:
> I don't play on my own server; I have never started up my own server, nor ever tried. To me seeing other people play or playing myself is to me way more fun. Also, I don't have much to test if I do start my own local server.
> 
> 2:
> This is a test thus maybe an excuse to add my own drip to the game. Everyone has to start somewhere with adding changes and this is a small thing, perfect for someone new to github branches. 
> 
> 3:
> I tried to add this with changing item icons, but the sad truth is that the icons are hardwired in the code using parent types for probably most clothing, including cloaks.

In that case, you might wish to change the changelog to administrative or such to not suffer the gbp loss of adding something unobtainable

**Comment by wesoda25** [comment]
Can you add a cloak for me while you're at it?

**Comment by Comxy** [comment]
> 
> 
> Can you add a cloak for me while you're at it?

That is honestly a good question too hard for me to answer. But I would like to say that it would be too hard for me to sprite and poll request a cape for every person as selfish as that may sound.

**Comment by MrDoomBringer** [comment]
walks in adds his own drip 
what a fucking based PR
merge it

**Comment by TheBonded** [comment]
make a check that gives you this cloak if you have your (@Comxy's) ckey so you always get it whenever you join a round

**Comment by Mothblocks** [comment]
this is honestly so powerful that i don't know if it's within my jurisdiction to close this

**Comment by D4C-420** [comment]
a 21 tc trade of a pr.

**Review by Krysonism — APPROVED** [review]
Sprite looks good!

**Comment by optimumtact** [comment]
An unobtainable cloak is not something we need, try to add it to a ruin, or make it in a vendor or anything.

**Comment by Mothblocks** [comment]
and then call it something other than ur ckey

**Comment by MrDoomBringer** [comment]
> and then call it something other than ur ckey

Some random cloak with worse stats then shift-start-available gear is not what this game needs
A commentary on the futility of player references, however? We could use that

**Comment by Timberpoes** [comment]
Also there is a way to var edit this to work in game. Someone will beat me to it but clothing has override vars for just this occasion. But I have definitely created custom var edited cloaks in the past.

**Comment by KittyNoodle** [comment]
Speedmerge this asap, the longer it takes the stronger he becomes

**Comment by LemonInTheDark** [comment]
I vote we toss him in a pit of lava

File: code/modules/clothing/suits/cloaks.dm [hunk 1]
```diff
@@ -48,6 +48,13 @@
 	name = "head of personnel's cloak"
 	desc = "Worn by the Head of Personnel. It smells faintly of bureaucracy."
 	icon_state = "hopcloak"
+	
+/obj/item/clothing/neck/cloak/comxy
```
Comment (Watermelon914) [inline code/modules/clothing/suits/cloaks.dm:52]: rename type to something other than comxy

File: code/modules/clothing/suits/cloaks.dm [hunk 2]
```diff
@@ -48,6 +48,13 @@
...
+	name = "drip cloak"
+	desc = "A cloak that makes you cooler than other people and it has a label called comxia on the inside of the cloak."
+	icon_state = "comxycloak"
```
Comment (Watermelon914) [inline code/modules/clothing/suits/cloaks.dm:55]: remove the player reference from the icon state and the description

**Comment by Comxy** [comment]
Alright, I tried reverting the initial name changes, since it checked the preupdated ones the drip objects was still called the comxy object, everything should be fine, I tried rereverting but I am not sure what is going on.

**Comment by Watermelon914** [comment]
please install visual studio code and use that to code instead of webedits. You need to properly test PRs to make sure they work.

**Comment by LemonInTheDark** [comment]
I swear to god if we add a drip cloak to match the drip jacket I'm going to break.

**Comment by Tennessee116** [comment]
This is the moment comxy became the master of the drip.

**Comment by Geevies** [comment]
oranges merge this

**Comment by maxymax13** [comment]
>adds his own clothing item
>does not elaborate
chad webeditor

**Comment by Comxy** [comment]
I will just close this before it auto closes. Personally I thought the sprite wasn't that good, so I see why it wasn't merged. The animation when the light disappeared didn't have a good transition. Also the recipe was probably too big, but I have learned from my mistakes, soon, a new one will open up one day sooner or later.

**Comment by LemonInTheDark** [comment]
It's like, we don't really like self references that are as clear as you're doing. Speaking personally I don't really like stuff like the drip outfit, think the aesthetic is a bit off.

**Comment by MrDoomBringer** [comment]
we were so close to greatness

**Pull request closed without merging**